### DISCUSSION TITLE
Cloudflare!!!

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6223] [General]
I tries many times to follow the documentation about cloudflare , not working , something wrong , what shoud i install first , also I dont need load balancer in the documentation ,  please i neeed you help ,  I follow the documentation as is manytime
something wrong , also there is a section in the AMS that says SSL ??

**Comment by burak-58** [comment]
Thank you for your question. Let me convert this to a github discussion so that community can follow the discussion.

**Comment by muratugureminoglu** [comment]
Hi @brixxnatt 

Are you stuck on step 1 or step 2? You should give more details so we can help you. Nginx is working here as a Reverse proxy, not as a load balancer. The parts with SSL are already explained in the documentation.
